spec/fixtures/extra_countries/en.yml

```diff
@@ -0,0 +1,7 @@
+SU:
```

Comment:
I don't think the extra directory is needed. Maybe just call it `spec/fixtures/extra_countries.yml`?